x-pack/plugins/osquery/public/packs/pack_queries_status_table.tsx

```diff
@@ -372,33 +372,31 @@ const ScheduledQueryLastResults: React.FC<ScheduledQueryLastResultsProps> = ({
     interval,
   });
 
+  const timestamp = useMemo(() => {
+    const dateTime = lastResultsData?.['@timestamp'];
+    if (!dateTime) return undefined;
+
+    return Array.isArray(dateTime) ? dateTime[0] : dateTime;
```

Comment:
It was reported that this issue is a regression. As a result, I’ve accounted for the possibility that some stack versions might return the search aggregation as a string rather than an array of strings.